### ISSUE TITLE
ATLAS ITk chain demonstrator

### ISSUE THREAD

**Issue by asalzburger** [acts-project/acts#1010]
With #999 being merged after CI finishes, we should have everything in hand to establish a first ITk chain demonstrator in `ActsExamples`.

Here's what I think is needed and particle missing, please tick the boxes/comment

- [ ] split configuration for the modules (@niermann999)
- [ ] material map, recorded in Athena, mapped in `ActsExamples) (@Corentin-Allaire)
- [ ] digitisation configuration smearing, and geometric (@niermann999 @asalzburger)
- [ ] seeding configuration (@noemina @LuisFelipeCoelho)

On top of those items, we should be able to run `ActsFatras - TruthTracking / Seeding - CKF` in several constellations.

**Comment by paulgessinger** [comment]
🥳

**Comment by Corentin-Allaire** [comment]
I don't think it would work to just transfert the map, for it to work the geometry Id would need to be exactly the same in both and I am not sure that is the case. Recording the material in athena then mapping it in Acts is much saffer and not too complicated (we just need to reformat the root file).

**Comment by noemina** [comment]
FYI: also the athena ITk Tracking geometry has been implemented in master and the material has been mapped. We can start comparing the configurations.

**Comment by paulgessinger** [comment]
> I don't think it would work to just transfert the map, for it to work the geometry Id would need to be exactly the same in both and I am not sure that is the case. Recording the material in athena then mapping it in Acts is much safer and not too complicated (we just need to reformat the root file).

That's exactly what I was thinking.

**Comment by noemina** [comment]
Reading the map is not what we should do, I agree, but remapping the same geantino steps on the ACTS tracking geometry (of course implementing the mapping procedure or using the same as done in ACTS standalone) is something we could do in athena as well, right?

**Comment by Corentin-Allaire** [comment]
Yeah, we will do the mapping separately in both Athena and Standalone but we can use the same material step as input and the same binning. The issue is that due to the way we automatically associate Id to layer and volumes the Id might slightly differ between Athena and Standalone so we cannot directly use the same map file but with some slight adjustment of the config file we could do the exact same mapping in both (and even compare them afterward).

**Comment by paulgessinger** [comment]
I think eventually, we will have to map onto the ACTS-ITk tracking geometry in Athena directly, without going through the standalone, because the material maps will not be compatible due to the geometry ids, yes. But with the tooling that @noemina has set up for the Athena-only material mapping now, it shouldn't be too hard to supply an algorithm that does the mapping.

**Comment by Corentin-Allaire** [comment]
Sorry maybe I was not clear we have everything needed to map the material onto the ACTS-ITk tracking geometry in Athena right now, I think by next week I will have it. Then we can redo it in standalone if needed.

**Comment by niermann999** [comment]
I opened a merge request for the splitting configuration on gitlab

**Comment by paulgessinger** [comment]
After discussion in the meeting, we're closing this for now.
